src/test/java/com/google/api/codegen/testsrc/common/samples/publish_series.sample.yaml

```diff
@@ -11,7 +11,7 @@ samples:
   region_tag: "canonical"
   request:
   - field: shelf.name
-    value: Math
+    value: "[Math]"
```

Comment:
I was confused by the brackets until I saw the config changed. I might have gone for the uuid field to make it less surprising, but no biggie. <shrug/>

Reply:
+1 that makes sense from a real user perspective ^ (uuid)

Reply:
The uuid is not handled by the same chunk of code though... `value` is the only thing that hates `[`.